pageserver/src/tenant.rs

```diff
@@ -3007,6 +3014,32 @@ impl Tenant {
                             .contains(CompactFlags::EnhancedGcBottomMostCompaction)
                         {
                             warn!("ignoring scheduled compaction task: scheduled task must be gc compaction: {:?}", next_scheduled_compaction_task.options);
+                        } else if next_scheduled_compaction_task.options.sub_compaction {
```

Comment:
Maybe call the flag `schedule_sub_compaction` instead?

Reply:
We do have two caller for subcompaction: either directly do it in the compaction API, or go through the scheduling.